server/base/src/main/java/org/apache/accumulo/server/manager/state/MetaDataStateStore.java

```diff
@@ -83,25 +91,30 @@ public void setFutureLocations(Collection<Assignment> assignments)
   }
 
   @Override
-  public void unassign(Collection<TabletLocationState> tablets,
+  public void unassign(ServerContext context, Collection<TabletLocationState> tablets,
```

Comment:
I think we probably don't need to keep passing in ServerContext to each instance method if we just add it to the constructor.

Reply:
I did that because the RootDataStateStore did not have access to the context as it was not constructed as such.

Reply:
Updated to construct the ZooTabletStateStore with the context.  Reset the TabletStateStore interface to what it was originally.